### PR TITLE
docs: Link to docs in README

### PR DESCRIPTION
**Pull request by rwwagner90**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,42 +2,17 @@
 
 # Ember-Electron
 
-![Latest release on NPM](https://img.shields.io/npm/v/ember-electron.svg) [![Build status](https://ci.appveyor.com/api/projects/status/5rhwhar361uad07v?svg=true)](https://ci.appveyor.com/project/adopted-ember-addons/ember-electron)
- ![Travis CI Build Status](https://secure.travis-ci.org/adopted-ember-addons/ember-electron.svg?branch=master) ![Code Climate](https://codeclimate.com/github/adopted-ember-addons/ember-electron.svg)
+![Latest release on NPM](https://img.shields.io/npm/v/ember-electron.svg) 
+[![Build status](https://ci.appveyor.com/api/projects/status/5rhwhar361uad07v?svg=true)](https://ci.appveyor.com/project/adopted-ember-addons/ember-electron)
```
Comment (bendemboski) [inline README.md]: Let's specify `Appveyor` here?

File: README.md [hunk 2]
```diff
@@ -2,42 +2,17 @@
...
+![Travis CI Build Status](https://secure.travis-ci.org/adopted-ember-addons/ember-electron.svg?branch=master) 
+![Code Climate](https://codeclimate.com/github/adopted-ember-addons/ember-electron.svg)
```
Comment (bendemboski) [inline README.md]: This 404s -- were you going to set up code climate?

File: README.md [hunk 3]
```diff
@@ -2,42 +2,17 @@
...
-
-- [Common issues](https://adopted-ember-addons.github.io/ember-electron/docs/faq/common-issues)
-- [File structure](https://adopted-ember-addons.github.io/ember-electron/docs/faq/structure)
-- [Security concerns](https://adopted-ember-addons.github.io/ember-electron/docs/faq/security)
-
-Somethings missing? Contributions to our docs are welcome!
+[View Docs](https://adopted-ember-addons.github.io/ember-electron/)
 
 
 ## Support
```
Comment (bendemboski) [inline README.md]: All this stuff down here is also duplicated in `index.md` -- how about we pick one or the other?

File: README.md (hunk 2 above)
Comment (rwwagner90) [reply README.md]: This was existing. We can remove, if you want.

File: README.md (hunk 3 above)
Comment (rwwagner90) [reply README.md]: I prefer not to. This is relevant to both spots.

File: README.md (hunk 1 above)
Comment (rwwagner90) [reply README.md]: Do what? I'm not familiar with Appveyor.

Comment (bendemboski) [reply README.md]: Appveyor is a CI system like travis, but for running windows builds. This link points to the appveyor build status, but the text of the link it `Build status`, so I'm just proposing we change it to say `Appveyor Build Status` (like we have `Travis Build Status` as the text of the travis link).

File: README.md (hunk 2 above)
Comment (bendemboski) [reply README.md]: Oh yeah, I see -- mis-read the diff. Well, let's pull it since it 404s.

**Pull request merged**